beacon_node/network/src/beacon_processor/mod.rs

```diff
@@ -898,7 +922,7 @@ impl<T: BeaconChainTypes> BeaconProcessor<T> {
         let mut status_queue = FifoQueue::new(MAX_STATUS_QUEUE_LEN);
         let mut bbrange_queue = FifoQueue::new(MAX_BLOCKS_BY_RANGE_QUEUE_LEN);
         let mut bbroots_queue = FifoQueue::new(MAX_BLOCKS_BY_ROOTS_QUEUE_LEN);
-
+        let mut lcbootstrap_queue = FifoQueue::new(MAX_BLOCKS_BY_ROOTS_QUEUE_LEN);
```

Comment:
we should have a different `const` for this, but maybe `1024` is good

Reply:
the request size in memory is really small anyway (32 bytes) but better to make another const variable.